app/views/shared/_header.html.haml

```diff
@@ -5,7 +5,7 @@
     %ul.col-md-4.clearfix
       %li= link_to "Videos"
       %li= link_to "My Queue", my_queue_path
-      %li= link_to "People"
+      %li= link_to "People", people_path
     = form_tag search_videos_path, class: "span 5 for-search" do
```

Comment:
`span 5` is not a valid class. You were going for `span-5` but thats not valid either. That was from Bootstrap 2. You need to use the new `col` style classes now.